tests/pipelines/kitctl/run.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipeline-template
+  namespace: tekton-pipelines
```

Comment:
Why are we running in this namespace? `tekton-pipelines` is the namespace where the tekton controller runs. Any reasons to not run in the default namespace?

Reply:
I think we should keep all the Tekton CRs in the same namespace for consistency. What do you think?